app/src/main/java/com/nextcloud/talk/chat/ChatActivity.kt

```diff
@@ -2144,17 +2144,6 @@ class ChatActivity :
         }
     }
 
-    private fun checkShowMessageInputView() {
-        if (isReadOnlyConversation() ||
-            shouldShowLobby() ||
-            !participantPermissions.hasChatPermission()
```

Comment:
these checks are now not used at all...
E.g.:
when "Lock conversation" is enabled on web, the android app now still shows the input field...

<img src=https://github.com/user-attachments/assets/5053114b-b418-40ee-a50d-790d22a967d7 width=250 />